### ISSUE TITLE
verifyJWT failing with "no DID resolver has been configured"

### ISSUE THREAD

**Issue by jordanmmck** [decentralized-identity/did-jwt#63]
Using the code snippets from the readme, `verifyJWT` fails with: `Error: No DID resolver has been configured`. Everything works up until step 2.

**To Reproduce**
1. Follow install instructions on readme
2. Paste code snippets from readme into index.js
3. `node index.js`

**Expected behavior**
Expect JWT to be verified.

**Desktop (please complete the following information):**
 - MacOS
 - Node v12.12.0
- Other deps:

```
    "did-jwt": "^3.0.0",
    "did-resolver": "0.0.7",
    "ethjs-provider-http": "^0.1.6",
    "ethr-did": "^1.1.0",
    "ethr-did-resolver": "^0.2.0",
    "web3": "^1.2.1"
```

**Comment by jordanmmck** [comment]
Thanks again @aldigjo. This line fails for me: `const ethrDid = require('ethr-did-resolver').getResolver();` with: `Uncaught TypeError: __webpack_require__(...).getResolver is not a function`. Appreciate the help.

**Comment by aldigjo** [comment]
@jordanmmck upgrade ethr-did-resolver to 1.0.2 and did-resolver to 1.0.0, this may the same problem on the other issue

**Comment by jordanmmck** [comment]
Those are the versions I'm using:

```
  "dependencies": {
    "did-jwt": "^3.0.0",
    "did-resolver": "^1.0.0",
    "ethjs-provider-http": "^0.1.6",
    "ethr-did": "^1.1.0",
    "ethr-did-resolver": "^1.0.2",
    "react": "^16.10.2",
    "react-dom": "^16.10.2",
    "react-scripts": "3.2.0",
    "web-did-resolver": "^1.0.2",
    "web3": "^1.2.1"
  },
```

**Comment by EvanTedesco** [comment]
I just began encountering this issue today on code that was working yesterday without issue.  I even checked out the commit to verify.  My dependencies are: 

`{
    "did-jwt": "^3.0.0",
    "did-resolver": "^1.1.0",
    "drizzle": "^1.4.0",
    "ethr-did-resolver": "^1.0.3",
    "react": "^16.12.0",
    "react-dom": "^16.12.0",
    "react-scripts": "3.2.0",
    "uport-connect": "^1.1.11"
  }`

**Comment by mirceanis** [comment]
This has been fixed in #57 and released in v4.0.0
A `resolver` is required when calling `verifyJWT`.

Please reopen if still relevant
